### DISCUSSION TITLE
OpenVPN Client

### DISCUSSION THREAD

**Discussion by katanza12** [indomus/forum#1495] [GENERALE - Discussioni generiche]
Ciao a tutti.
Stavo seguendo la guida indomus per abilitare l'accesso da remoto ad HA core con provider che hanno gli ip nattati (Sky WiFi nel mio caso). Mi sono bloccato però nello step dove bisogna [configurare il client vpn sul raspberry](https://indomus.it/guide/collegare-raspberry-pi-a-una-vpn-con-raspberry-pi-os/). In particolare, quando c'è da avviare il servizio con il comando:
```
sudo systemctl start openvpn@NOME_UENTE.service
```
Non ho capito con cosa bisogna sostituire NOME_UTENTE. E se provo ad usare quello utilizzato in precedenza nella guida (openvpn-client) mi da questo errore:
```
Options error: In [CMD-LINE]:1: Error opening configuration file: /etc/openvpn/openvpn-client.conf
```
Perchè effettivamente quel file non esiste e non c'è nessun passo nella guida per crearlo.

Ho cercato anche in rete possibili soluzioni, ma tutto quello che ho trovato è profondamente diverso dai passi precedenti della guida di indomus, vorrei evitare quindi di appiccicare insieme pezzi diversi che non solo non mi farebbero arrivare ad una soluzione, ma mi riempirebbero di "porcheria" il sistema. 

Qualcuno ha già affrontato questo problema? Ho ha una guida alternativa ed affidabile per collegare il raspberry ad una vpn fatta su google vm con debian e OpenVPN Server istallato?

Grazie

**Comment by KC-inDomus** [comment]
"Come anticipato, assumeremo che la VPN alla quale andremo a collegare il nostro Raspberry sia basata su OpenVPN.
Ciò che ci servirà è:

il file (con estensione .ovpn) contenente le chiavi dell’utente configurato lato server OpenVPN;
la password relativa all’utente configurato lato server OpenVPN.
Questa informazioni, quindi, si reperiscono lato server.

Si assume altresì che il Raspberry Pi sia configurato (inclusa la possibilità di collegarvisi tramite SSH), collegato alla rete locale e, attraverso essa, a Internet."

hai rispettato questi assunti?

**Comment by katanza12** [comment]
Si, assolutamente. Tutti gli step precedenti li ho fatti senza problemi

Roberto Catania
Inviato da iPhone
